xmodule/tests/test_course_block.py

```diff
@@ -288,6 +288,7 @@ def test_default_discussion_topics(self):
         assert {'General': {'id': 'i4x-test_org-test_course-course-test'}} == d.discussion_topics
 
 
+@patch("openedx.core.lib.teams_config.CONTENT_GROUPS_FOR_TEAMS.is_enabled", lambda _: False)
```

Comment:
Note to reviewer: we still need to add unit tests for this approach, but we'll do it once it is accepted.